internal/runners/config/config.go

```diff
@@ -18,13 +23,103 @@ type primeable interface {
 	primer.Analyticer
 }
 
-func NewConfig(prime primeable) (*Config, error) {
-	return &Config{
+func NewList(prime primeable) (*List, error) {
+	return &List{
 		out: prime.Output(),
 		cfg: prime.Config(),
 	}, nil
 }
 
-func (c *Config) Run(usageFunc func() error) error {
-	return usageFunc()
+type configData struct {
+	Key     string `locale:"key,Key"`
+	Value   string `locale:"value,Value"`
+	Default string `locale:"default,Default"`
```

Comment:
We've started moving away from magical output handling, please see how I handled table layout here:

https://github.com/ActiveState/cli/blob/d43bc4921ea8fccf7f1848f4ba2e22004121e145/internal/runners/upgrade/upgrade.go#L248-L248

Basically it's just manually constructing the table and rendering the resulting string itself.

Reply:
Also please add json keys, right now json output is capitalized.